cmd/openscap-plugin/oscap/oscap.go

```diff
@@ -32,26 +58,41 @@ func constructScanCommand(openscapFiles map[string]string, profile string) []str
 func OscapScan(openscapFiles map[string]string, profile string) ([]byte, error) {
 	command := constructScanCommand(openscapFiles, profile)
 
-	cmdPath, err := exec.LookPath(command[0])
-	if err != nil {
-		return nil, fmt.Errorf("command not found: %s", command[0])
+	return executeCommand(command)
+}
+
+func constructGenerateFixCommand(fixType string, output string, profile string, tailoringFile string, datastream string) []string {
```

Comment:
Super tiny nit: Since all the parameters share the same type, you only need to specify the type once.
```suggestion
func constructGenerateFixCommand(fixType, output, profile, tailoringFile, datastream string) []string {
```